pom.xml

```diff
@@ -371,48 +370,22 @@
     <pluginManagement>
       <plugins>
         <plugin>
+          <!-- TODO remove with next parent -->
           <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-shade-plugin</artifactId>
-          <version>3.2.4</version><!-- TODO remove when upgrading parent to 35 -->
+          <artifactId>maven-pmd-plugin</artifactId>
+          <version>3.16.0</version>
+        </plugin>
+        <plugin>
+          <!-- TODO remove with next parent -->
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>taglist-maven-plugin</artifactId>
+          <version>3.0.0</version>
         </plugin>
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
-          <executions>
```

Comment:
@slawekjaranowski 
I do not know why you have removed this part. The main purpose was `<arg>-Xdoclint:all</arg>` and we bypass the issue with generated classes HelpMojo.

Reply:
`<arg>-Xdoclint:all</arg>` still is in configuration line: 392

Reply:
`HelpMojo` now has correct javadocs - we needn't exclude for it

Reply:
@slawekjaranowski 
Do you have `<arg>-Xdoclint:all</arg>` in your commit? Is it in MPOM? The point is to get the error within compile time because this is what the contributors usualy do. They do not run `mvn site`. All of them run the normal build as they have experiences.